lsp-mode.el

```diff
@@ -4659,14 +4659,74 @@ Stolen from `org-copy-visible'."
 
     (lsp--setup-markdown major-mode)))
 
+(defvar lsp--display-inline-image-alist
+  '((lsp--render-markdown
```

Comment:
Instead of mode, function alist, I think this can be just a list of functions (list of current cdr of each item now). The reason is that those markdowns are not scoped to a certain mode.
Also, do you want to contribute this to markdown mode too?

Reply:
@kiennq 
> Shouldn't this be latex-mode instead?

No, as far as I know. The function in `lsp--fontlock-with-mode` render not buffer edited by user, but content sent from lsp server (because it is only called by `lsp--render-string`). So, the second argument `mode` express language in which lsp server write contents. 
In the example case on lsp-latex, while major-mode of buffer edited by user is `latex-mode`, the argument `mode` is `lsp--render-markdown`, because texlab, the lsp server for latex, send contents with "markdown".

>Instead of mode, function alist, I think this can be just a list of functions (list of current cdr of each item now). The reason is that those markdowns are not scoped to a certain mode.

This regexp is only for replacing markdown-styled link to image, so it should be separated by modes. In the future, some image displayer might be added on mode in `lsp-language-id-configuration`, such as html.


>Also, do you want to contribute this to markdown mode too?

Because I don't usually use markdown-mode, I'm not eager, but willing.

Reply:
Thanks